app/models/user.rb

```diff
@@ -5,4 +5,7 @@ class User < ApplicationRecord
   def recent_posts(limit = 3)
     posts.order(created_at: :desc).limit(limit)
   end
+
+  validates :name, presence: true
```

Comment:
- [ ] Kindly ensure that the name validation is tested for the case when a user does not enter a name. 👌😊